src/types/UserLoginProps.ts

```diff
@@ -0,0 +1,10 @@
+interface UserLoginProps {
+    id: string;
+    password: string;
+    email?: UserEmail;
```

Comment:
Not sure why not consider directly as a string the email, it's great if we plan to add something else later to that type tho.